components/button/button.css

```diff
@@ -0,0 +1,73 @@
+body {
+  font-family: var(--nunito);
+}
+
+.solid-btn {
+  border: none;
+  border-radius: 0.4rem;
+}
+
+.outline-btn {
+  border: 2px solid;
+  border-radius: 0.4rem;
+  background-color: var(--white);
+}
+
+.icon-btn {
+  display: flex;
+  border-radius: 0.4rem;
+  border: none;
```

Comment:
instead of writing border-radius and border none at different instance, you can create a class for it and use it at different instance

Reply:
I'hv used border property only once in this component, i'll add a class if needed.